.prettierignore

```diff
@@ -2,10 +2,6 @@ dist/
 lib/
 coverage/
 
-fixtures/
-public/
-types/
```

Comment:
We don't have any directories called `public` or `types` that we want to ignore, so we don't need this in the config.